joomla/joomla-cms
# [4.0] Remove link to view that has been removed

### Summary of Changes
Remove the dashboard link to purge the cache as this view has been removed a while ago.

### Testing Instructions
1. Go to System
2. Notice the link to remove expired cache, clicking this will throw an error because the view has been removed
![image](https://user-images.githubusercontent.com/359377/64621102-652e1080-d3e5-11e9-8492-d4edbdf3ed33.png)
3. Apply PR
4. Reload page
5. Notice the link is gone
![image](https://user-images.githubusercontent.com/359377/64621049-4cbdf600-d3e5-11e9-83cb-79f44bc74026.png)


### Expected result
There is no link because there is no view


### Actual result
There is a link that throws an error


### Documentation Changes Required
None
